bootstrap/gulpfile.js

```diff
@@ -1,16 +1,18 @@
-var gulp  = require('gulp'),
-  sass = require('gulp-sass'),
-  sourcemaps = require('gulp-sourcemaps'),
-  cleanCss = require('gulp-clean-css'),
-  rename = require('gulp-rename'),
-  postcss      = require('gulp-postcss'),
-  autoprefixer = require('autoprefixer');
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var sourcemaps = require('gulp-sourcemaps');
+var cleanCss = require('gulp-clean-css');
+var rename = require('gulp-rename');
+var postcss = require('gulp-postcss');
+var autoprefixer = require('autoprefixer');
```

Comment:
Are the above changes necessary? All versions of JS I've ever used allow grouping variable declarations in this way...

Reply:
I don't think they're really necessary.